README.md

```diff
@@ -117,7 +149,7 @@ Alfred Workflow with dozens of features for controlling your [Obsidian vault](ht
 ## About the Developer
 In my day job, I am a sociologist studying the social mechanisms underlying the digital economy. For my PhD project, I investigate the governance of the app economy and how software ecosystems manage the tension between innovation and compatibility. If you are interested in this subject, feel free to get in touch.
 
-__Profiles__  
```

Comment:
These are double-trailing spaces, which are meaningful in markdown, please restore them ("two-space-rule")